src/server/accounting.c

```diff
@@ -1083,6 +1083,7 @@ account_jobend(job *pjob, char *used, int type)
 	char *resc_used;
 	int resc_used_size = 0;
 	pbs_list_head temp_head;
+	int val_len = 0;
```

Comment:
Looks like a consequence of a cherry-pick and a later feature checkin that took out val_len. I'll remove this unnecessary declaration.